src/brown-sauce.md

```diff
@@ -0,0 +1,31 @@
+# Brown Sauce
+
+A very basic sauce that can be used for various dishes.
+
+- ⏲️ Prep time: 5 min
+- 🍳 Cook time: 20 min
+- 🍽️ Servings: 6
+
+## Ingredients
+
+- 1 onion
+- 75g butter
+- 1dl flour
+- 1l bouillon
+- Black pepper freshly grounded
```

Comment:
Salt/Pepper shouldn't be added as ingredients. You can keep it in the directions below though.